### DISCUSSION TITLE
Can't get xaml intellisense

### DISCUSSION THREAD

**Discussion by meleeeee** [unoplatform/uno#8963] [Q&A]
Hello, 
I had a good solution here before. Thanks for the quick reply.

Now  I'm making library for user controls with default template.
> vs2019 16.11.15 version
> 'Uno Platform Library (Xamarin)' template

And I'd like to develop user controls target net5.0 (because I need this library for webassembly project)
So I only need net5, but if I set it like below, the Xaml Designer is not visible.
`<TargetFrameworks>net5.0</TargetFrameworks>`

I finally found through a search that this was due to the target framework, and I add uap like this.
`<TargetFrameworks>uap10.0.19041;net5.0</TargetFrameworks>`

But this was not the solution. There is Designer now but it said,
**"Some assembly references are missing"**

Any ideas on how to solve this?
I'll attach the whole source of .csproj just in case.
(and just in case, I added netstandard2.0 too, but the designer was gone)

```
<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
  <!--
	Adding project references to this project requires some manual adjustments.
	Please see https://github.com/unoplatform/uno/issues/3909 for more details.
	-->
  <PropertyGroup>
    <TargetFrameworks>uap10.0.19041;net5.0</TargetFrameworks>
    <!-- Ensures the .xr.xml files are generated in a proper layout folder -->
    <GenerateLibraryLayout>true</GenerateLibraryLayout>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Uno.UI" Version="4.3.8" />
  </ItemGroup>
  <ItemGroup>
    <Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
    <Compile Update="**\*.xaml.cs">
      <DependentUpon>%(Filename)</DependentUpon>
    </Compile>
  </ItemGroup>
  <ItemGroup>
    <UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
  </ItemGroup>
  <ItemGroup>
    <None Remove="testclock.xaml" />
  </ItemGroup>
  <ItemGroup>
    <UpToDateCheckInput Remove="testclock.xaml" />
  </ItemGroup>
  <ItemGroup>
    <Compile Update="testclock.xaml.cs">
      <DependentUpon>testclock.xaml</DependentUpon>
    </Compile>
  </ItemGroup>
</Project>
```

**Comment by jeromelaban** [comment]
Unfortunately, at this time, the designer is not supported in this type of class libraries. Microsoft never supported designing UIs using "SDK-style projects" with UWP, and this is causing the issue you're seeing.

One way to work around this is to create an "original" UAP csproj (called `MyProjectLibrary.Designer`, for instance) and place it in the same folder as the project library you already have. This way, you can design using `MyProjectLibrary.Designer`, and build using `MyProjectLibrary`.

Note that using .NET 6 and WinAppSDK, there is no designer either, but this time because Microsoft is not providing one yet, but that may change in the future.
